downloads/migrations/0014_releasefile_sha256_sum.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.26 on 2025-11-27 16:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('downloads', '0013_alter_release_content_markup_type'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='releasefile',
+            name='sha256_sum',
+            field=models.CharField(blank=True, max_length=200, verbose_name='SHA256 Sum'),
```

Comment:
Per above, if relevant
```suggestion
            field=models.CharField(blank=True, max_length=200, verbose_name='SHA256 Checksum'),
```